test/e2e/common/container_probe.go

```diff
@@ -82,9 +82,10 @@ var _ = framework.KubeDescribe("Probing container", func() {
 	})
 
 	/*
-		    Testname: pods-readiness-probe-failure
-		    Description: Make sure that pod with readiness probe that fails should
-			never be ready and never restart.
+		Release : v1.9
+		Testname: Pod readiness probe failure
+		Description: Pod is created with a readiness probe that fails consistently. When this Pod is created,
+			then the Pod SHOULD never be ready, never be running and restart count SHOULD be zero.
```

Comment:
SHOULD aka "there may exist valid reasons in particular circumstances to ignore a particular item"

should this be MUST? what am I missing that would make it be a SHOULD?

Reply:
it should be MUST, fixed.